filters/auth/oidc_test.go

```diff
@@ -864,6 +864,7 @@ func TestOIDCSetup(t *testing.T) {
 			backend := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 				requestDump, _ := httputil.DumpRequest(r, false)
 				assert.Contains(t, string(requestDump), tc.expectRequest, "expected request not fulfilled")
+				assert.NotContains(t, string(requestDump), oauthOidcCookieName, oauthOidcCookieName+" cookie should be dropped")
```

Comment:
Please extend or add a test to verify that _other_ cookies are passed es expected